artemis-protocols/artemis-mqtt-protocol/src/main/java/org/apache/activemq/artemis/core/protocol/mqtt/MQTTUtil.java

```diff
@@ -372,7 +374,7 @@ public static void logMessage(MQTTSessionState state, MqttMessage message, boole
                   break;
             }
 
-            logger.trace(log.toString());
+            logger.info(log.toString());
```

Comment:
Seems a bit spammy, leftover debug change? EDIT Also the whole method is wrapped in an 'isTraceEnabled' gate.